kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -1,38 +1,92 @@
-    <span class="current">
-      <span class="visuallyhidden">Current: </span>
-        <!-- TODO: Get current topic title -->
-    </span>
-  </nav>
+  <div>
+    <nav class="nav" role="navigation" aria-label="You are here:">
+      <span class="parent full-breadcrumbs" v-if="pageMode === $options.PageModes.EXPLORE && pageName !== $options.PageNames.EXPLORE_CONTENT && !isRoot">
+        <a v-link="exploreRoot">Explore</a>
+      </span>
+      <span v-if="pageName === $options.PageNames.LEARN_CONTENT">
+        <a v-link="learnRoot">
+          <span class='sep'>
+            <svg height="24" viewbox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"></path>
+            </svg>
+          Learn
+          </span>
+        </a>
+      </span>
+      <span v-if="pageName === $options.PageNames.EXPLORE_CONTENT">
+        <a v-link="parentLink">
+          <span class='sep'>
+            <svg height="24" viewbox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"></path>
+            </svg>
+          Back
+          </span>
+        </a>
+      </span>
+      <span class="parent full-breadcrumbs" v-if="pageMode === $options.PageModes.EXPLORE" v-for="crumb in crumbs">
+        <a v-link="crumbLink(crumb.id)">{{ crumb.title }}</a>
+      </span>
+      <span class="back" v-if="!isRoot && pageMode === $options.PageModes.EXPLORE && pageName !== $options.PageNames.EXPLORE_CONTENT">
+        <a v-link="backLink">
+          <span class='sep'>
+            <svg height="24" viewbox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+              <path d="M0 0h24v24H0z" fill="none"></path>
+              <path d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"></path>
+            </svg>
+          Back
+          </span>
+        </a>
+      </span>
+    </nav>
+  </div>
 
 </template>
 
 
 <script>
 
   const PageNames = require('../../state/constants').PageNames;
+  const constants = require('../../state/constants');
+  const getters = require('../../state/getters');
 
   module.exports = {
-    props: {
-      rootid: {
-        type: String,
-        required: true,
+    mixins: [constants], // makes constants available in $options
+    computed: {
+      learnRoot() {
+        return { name: PageNames.LEARN_ROOT };
       },
-      crumbs: {
-        type: Array,
-        required: true,
+      exploreRoot() {
+        return { name: PageNames.EXPLORE_ROOT };
       },
-    },
-    computed: {
-      rootLink() {
+      parentLink() {
+        let bread;
+        let id;
+        if (this.pageState.content) {
+          bread = this.pageState.content.breadcrumbs;
+          id = bread[bread.length - 1].id;
+        }
```

Comment:
what if someone changes how the pageName is structured? also the breadcrumbs property is stored inside pageState, that's where the id came from.